spec/bgs/services/benefit_claims_spec.rb

```diff
@@ -48,4 +48,18 @@
       expect(response[:bnft_claim_dto][:bnft_claim_id]).to eq(test_claim_id)
     end
   end
+
+  it 'updates benefit claim' do
+    VCR.use_cassette('benefit_claims/update_bnft_claim') do
+      claim = service.benefit_claims.find_bnft_claim(claim_id: test_claim_id)[:bnft_claim_dto]
```

Comment:
if the above is true, we wouldn't need the last part of this ( `[:bnft_claim_dto]` )